data-reporting-service/investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java

```diff
@@ -44,10 +52,39 @@ public InvestigationTransformed transformInvestigationData(Investigation investi
         transformNotificationIds(investigation.getObservationNotificationIds(), objectMapper);
         transformObservationIds(investigation.getObservationNotificationIds(), investigationTransformed, objectMapper);
         transformInvestigationConfirmationMethod(investigation.getInvestigationConfirmationMethod(), investigationTransformed, objectMapper);
+        processInvestigationPageCaseAnswer(investigation.getInvestigationCaseAnswer(), objectMapper);
```

Comment:
Page case answers bypass the connectors while notfications will still use the Kafka topics/Sink connector path right.

Reply:
right!